tests/cucumber/features/authentication/authentication.feature

```diff
@@ -7,3 +6,13 @@ Feature: Login
   Background:
     Given I am on the home page
     Given I am signed out
+
+  @dev
+  Scenario: Create account
+    Given "test@test.com" has not registered
```

Comment:
The funny thing is that when i run `meteor mongo` and check the users collection, there is nothing there.